sandbox/StatusInputPage.qml

```diff
@@ -89,14 +89,15 @@ Column {
         input.placeholderText: "Input with validator"
 
         validators: [
-            StatusMinLengthValidator { minLength: 10 }
-        ]
-
-        onTextChanged: {
-            if (errors && errors.minLength) {
-                errorMessage = `Value can't be shorter than ${errors.minLength.min} but got ${errors.minLength.actual}`
+            StatusMinLengthValidator {
+                minLength: 10
+                error: {
+                    if (errors && errors.minLength) {
+                        errorMessage = `Value can't be shorter than ${errors.minLength.min} but got ${errors.minLength.actual}`
+                    }
+                }
```

Comment:
This is interesting.
As mentioned in: https://github.com/status-im/StatusQ/commit/ba4f27f9ba6e957908d743c571ebc15f2399cd71, error messages have been pushed out of the validator pipeline, so applications can decide how they want to render them (especially needed when there's custom i18n solutions).

But it seems you've provided an API so that validators **do** provide a default error message, but applications still get a way to override it (as seen here).

I like that.